test/unittest/blas1/blas1_copy_test.cpp

```diff
@@ -26,14 +26,15 @@
 #include "blas_test.hpp"
 
 template <typename scalar_t>
-using combination_t = std::tuple<int, int, int>;
+using combination_t = std::tuple<int, int, int, scalar_t>;
```

Comment:
Can you add a brief comment here about the reason why this change was required? (along with the `llvm` issue being tracked on their Github repo). That will be helpful when we try this fix for other operators later on.

Reply:
Do you have the link the llvm issue?

Reply:
https://github.com/intel/llvm/issues/7075